src/singular.js

```diff
@@ -20,7 +27,8 @@ var Singular = {
     checkInterval: 1000,
     uaaLocation: '',
     storageKey: 'singularUserIdentityClaims',
-    authTimeout: 20000
+    authTimeout: 20000,
+    domContentLoaded: false
```

Comment:
This relates to the conditional trigger in the init function. There may be several different use cases for attaching the frames on init instead of attaching an event listener for `DOMContentLoaded`. If we are providing this functionality, it should probably cover generic uses cases (after making some AJAX calls, or getting/setting some state, for example) - another property name might be more appropriate.

A corresponding entry in the README.md file should be added.

Reply:
We will rename the property to `appendFramesOnInit`.